modules/geo-layers/src/tile-layer/tileset-2d.js

```diff
@@ -264,7 +270,7 @@ export default class Tileset2D {
     const tileId = `${x},${y},${z}`;
     let tile = this._cache.get(tileId);
 
-    if (!tile && create) {
+    if ((!tile || tile.isCancelled) && create) {
```

Comment:
You shouldn't need to create a new instance. Just call `loadData` again.

Reply:
Good point